packages/neuron-ui/src/components/Amount/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { UANTonkenSymbol } from 'components/UANDisplay'
+
+import { shannonToCKBFormatter, sudtValueToAmount, sUDTAmountFormatter, nftFormatter } from 'utils'
+import { HIDE_BALANCE } from 'utils/const'
+
+import styles from './overview.module.scss'
+
+export const Amount = ({ item, show }: { item: State.Transaction; show: boolean }) => {
```

Comment:
The name is too generic, it can be more precise with a name like `FormattedTokenAmount` or something similar.